buildpatches/dev_cel_expr.patch

```diff
@@ -1,14 +1,15 @@
-diff proto/cel/expr/BUILD.bazel proto/cel/expr/BUILD.bazel
+diff --git a/proto/cel/expr/BUILD.bazel b/proto/cel/expr/BUILD.bazel
+index d80de43..29db706 100644
 --- proto/cel/expr/BUILD.bazel
 +++ proto/cel/expr/BUILD.bazel
-@@ -70,5 +70,10 @@ go_proto_library(
-         ":syntax_proto",
-         ":value_proto",
+@@ -72,8 +72,8 @@ go_proto_library(
      ],
-+    deps = [
-+        "@org_golang_google_genproto_googleapis_rpc//status",
+     visibility = ["//visibility:public"],
+     deps = [
+-        ":syntax_go_proto",
+-        ":value_go_proto",
+         "@org_golang_google_genproto_googleapis_rpc//status",
 +        "@org_golang_google_protobuf//reflect/protoreflect",
 +        "@org_golang_google_protobuf//runtime/protoimpl",
-+    ],
```

Comment:
It seems like the new Gazelle version is picking up some wrong target here @fmeum 

The patched target looks like this
```
go_proto_library(
    name = "expr_go_proto",
    importpath = "cel.dev/expr",
    protos = [
        ":checked_proto",
        ":eval_proto",
        ":explain_proto",
        ":syntax_proto",
        ":value_proto",
    ],
    visibility = ["//visibility:public"],
    deps = [
        "@org_golang_google_genproto_googleapis_rpc//status",
        "@org_golang_google_protobuf//reflect/protoreflect",
        "@org_golang_google_protobuf//runtime/protoimpl",
    ],
)
```

the `syntax_go_proto` and `value_go_proto` does not exist in the BUILD file. They should already be included as part of `protos` attribute.

Reply:
I'll run a bisect to figure out what's up with this.

Reply:
I'm still not sure what's happening here. The references that newly pop up here are caused by some unnecessary directives (see https://github.com/buildbuddy-io/buildbuddy/pull/8186). But even with that change applied, I get unstable results after the update.

Reply:
Okay, I finally found the culprit: https://github.com/bazel-contrib/bazel-gazelle/commit/27bbbc6d1d06d5c8c6b7aafe7c46010a0e547309

This was hard to bisect due to the staleness issue only fixed in Gazelle 0.41.0.

Reply:
I pushed a new commit that simplifies the config. The "culprit" is a bugfix that gets this target to build without any patches.